### ISSUE TITLE
[SSO Request]  Support for  Active Directory Federation Services (ADFS)

### ISSUE THREAD

**Issue by infoseek1** [sandstorm-io/sandstorm#702]
Active Directory Federation Services (ADFS) is a software component developed by Microsoft that can be installed on Windows Server operating systems to provide users with single sign-on access to systems and applications located across organizational boundaries. It uses a claims-based access control authorization model to maintain application security and implement federated identity.

**Comment by ocdtrekkie** [comment]
Yeah, AD/LDAP support is a common request. They know, I bug them about it. :)

**Comment by kentonv** [comment]
This is a little different from the usual AD/LDAP -- they want ADFS login to e.g. Oasis. Apparently ADFS can be a global identity source, but we'll have to investigate the details.
